### ISSUE TITLE
Undefined variable: te

### ISSUE THREAD

**Issue by ValenokPC** [duzun/hQuery.php#25]
Pri zaprose `$data = hQuery::fromUrl("https://www.google.ru/");`

Poluchaju vot takie preduprezhdenija!

```
Notice:  Undefined variable: te in ...\duzun\hquery\hquery.php on line 2907

Notice:  Undefined variable: te in ...\duzun\hquery\hquery.php on line 2909

Notice:  Undefined variable: te in ...\duzun\hquery\hquery.php on line 2915
```

**Comment by duzun** [comment]
Fixed warnings with [#ea9cf95cb](https://github.com/duzun/hQuery.php/commit/ea9cf95cb78d58d2a2dda5bf1b31d45ea3395bf6)

**Comment by ValenokPC** [comment]
Thank you!
